app/model/album.js

```diff
@@ -0,0 +1,43 @@
+var express = require('express');
+var app = express();
+var fs =require('fs');
+
+Album = function() {
+  this.albums = [];
+  this.loadAll();
+}
+
+Album.prototype.get =function(id) {
+  if (!id) {return false;}
```

Comment:
Consider adding whitespace to increase readability for the purpose of maintainability.